src/Shared/FileUtilities.cs

```diff
@@ -59,19 +67,16 @@ static internal partial class FileUtilities
         private static void GetTestExecutionInfo()
```

Comment:
Nit: this could be renamed to be more accurate now that it checks for VS too. `GetExecutionInfo`?